src/status_im/ui/screens/wallet/assets/views.cljs

```diff
@@ -14,42 +11,49 @@
             [status-im.ui.screens.wallet.assets.styles :as assets.styles]
             [status-im.ui.screens.wallet.main.styles :as main.styles]
             [status-im.ui.components.tabs.views :as tabs]
-            [status-im.ui.components.list.views :as list]
-            [status-im.ui.components.styles :as components.styles]))
+            [status-im.ui.components.styles :as components.styles]
+            [status-im.ui.screens.wallet.transactions.views :as transactions]
+            [status-im.utils.utils :as utils]
+            [status-im.ui.components.chat-icon.styles :as chat-icon.styles]
+            [status-im.utils.money :as money]))
 
 
 (defview my-token-tab-title [active?]
-  (letsubs [ {:keys [token-symbol]} [:token-balance]]
+  (letsubs [ {:keys [symbol]} [:token-balance]]
```

Comment:
Hm, if we can avoid shadowing a core fn (symbol) that'd be good IMO. Is there something wrong with calling it `token-symbol`? Aside from the repetition. Or maybe there's a different name we can use?

Reply:
It was `token-symbol` in the mocked code. This rename is to conform with the map with all token info. In general, I agree with this - too bad all good (i.e generic var names) are already taken by core.